groups/promo-infrastructure/profiles/heritage-staging-eu-west-2/vars

```diff
@@ -0,0 +1,12 @@
+# Account details
```

Comment:
There should be no heritage profiles in this folder as its going to be Shared Services only.

Reply:
This is now removed as it has it's own group.